back/db/migrate/20221111132019_create_page_projects_filter_join_tables.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreatePageProjectsFilterJoinTables < ActiveRecord::Migration[6.1]
+  def change
+    create_table :static_pages_topics, id: :uuid do |t|
+      t.references :topic, index: true, foreign_key: true, type: :uuid
+      t.references :static_page, index: true, foreign_key: true, type: :uuid
+
+      t.timestamps
+    end
+
+    create_table :areas_static_pages do |t|
```

Comment:
Why do we reverse the order? `static_pages_topics` vs. `areas_static_pages`.

I know we can use either order for a single join table, but why not be consistent?

Reply:
I guess it's because we already have a number of similar join tables beginning with `areas_`. In which case, we could switch to `topics_static_pages`.

But, now I see the variation in existing conventions, I don't feel strongly about this.